<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Scanning</string>
    </property>
    <widget class="QPushButton" name="pushButton_Load_image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load image</string>
     </property>
    </widget>
    <widget class="QLabel" name="QPixmap_ui">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Hide">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Hide</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Triangles">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute triangles</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Path">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Signals">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute signals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Go">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_iterations">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>40000</number>
     </property>
     <property name="singleStep">
      <number>4</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Iterations :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_time_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Required time :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
